docs/decisions/0001-API-strategy-and-change-management-process.rst

```diff
@@ -0,0 +1,62 @@
+API Strategy and Change Management Process
+=============================================
+
+Status
+------
+
+Draft
+
+
+Context
+-------
+
+As enterprise APIs evolve, we need to ensure that changes are introduced in a backward-compatible manner that doesn't break existing clients, and that we have a clear and consistent process for handling these changes.
+
+Solution Approach
+-----------------
+
+By adopting an API change management process, we can ensure that our API evolves in a controlled and predictable manner. This approach will ensure minimal disruptions and provide a clear approach on how to handle upstream changes to Enterprise APIs. The API change management process includes the following components:
+
+- Guidelines around API versioning
+- Guidelines around dealing with breaking changes
+- Communicating upstream changes internally and externally
+
+
+Decision
+--------
+
+**API Versioning**
+
+- Currently, we use URI versioning in all external and internal APIs. We will continue to use this approach as it's simple and doesn't require a lot of customization on client's end. Begin with version (v1) when the API is initially released.
```

Comment:
we can also use semantic versioning, which provides a standard method to communicate breaking changes

Reply:
It's a nice suggestion. I think we can follow semantic versioning in docs..